cyral/model_integration_logging.go

```diff
@@ -94,7 +94,7 @@ func getIntegrationLogsSchema() map[string]*schema.Schema {
 			Default:       false,
 			ConflictsWith: []string{FluentbitKey},
 		},
-		"cloud_watch": {
+		"cloudwatch": {
```

Comment:
Use the const `CloudWatchKey` here instead.

Reply:
Addr here: [c8f4bb8](https://github.com/cyralinc/terraform-provider-cyral/pull/400/commits/c8f4bb8547bfd8b49604637c8dafa9b2f3474096). 

Thanks for splitting the examples!